Sources/SentrySwift/SwiftDescriptor.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+@objc
+public class SwiftDescriptor: NSObject {
+ 
+    @objc
+    public static func getDescription(_ object: AnyObject) -> String {
```

Comment:
`h`: I think this must not be public.
```suggestion
class SwiftDescriptor: NSObject {
 
    @objc
    static func getDescription(_ object: AnyObject) -> String {
```

Reply:
This need to be public in order to Objc to use it.
Swift code belongs in a different module, therefore it needs to be public.

But, to use this in a project, you need to import SentrySwift

Reply:
I removed the `public`, and it compiled. Does the compilation then only fail for the package managers? This class shouldn't be public. Nobody has to use this.

> But, to use this in a project, you need to import SentrySwift

So if we add actual public Swift code now, the user needs to add a different import? I think that is not good. I would like to avoid that.

Reply:
SPM fails if you remove the public.

> So if we add actual public Swift code now, the user needs to add a different import? I think that is not good. I would like to avoid that.

Thats why Swift will be only available to help us. But we can't create public api in Swift. I think we could make swift code available through `#import <Sentry/Sentry.h>` but then everything in swift will be public.

Reply:
Hummm, For cocoa and Carthage everything in Swift will be public already 😕 for this approach

Reply:
As discussed in the stand-up, I think we should try to avoid this.

Reply:
The solution will be to remove this Class from Sentry module, then, when a user import Sentry, this will not show in the auto complete. 

It's not possible to remove the public from this class and functions.

Reply:
@brustolin, this is the only issue stopping me from approving the PR. Did you already find a solution to it? How can I support you?